packages/next/src/build/webpack-config.ts

```diff
@@ -1121,6 +1109,11 @@ export default async function getBaseWebpackConfig(
   const optOutBundlingPackages = EXTERNAL_PACKAGES.concat(
     ...(config.experimental.serverComponentsExternalPackages || [])
   )
+  const optOutBundlingPackageRegex = new RegExp(
+    `[/\\\\]node_modules[/\\\\](${optOutBundlingPackages
+      .map((p) => p.replace(/\//g, '[/\\\\]'))
+      .join('|')})[/\\\\]`
+  )
```

Comment:
Using RegExp as loader conditions (that runs over **all** imports) should be faster than function calls with multiple `includes` when scaled up.